test/test_autograd.py

```diff
@@ -5812,7 +5836,62 @@ def foo(x, y):
         gradcheck(foo, inputs)
         gradgradcheck(foo, inputs)
 
-    def test_hessian_err_check(self):
+    def test_jacobian_create_graph(self):
+        self._test_jacobian_create_graph(vectorize=False)
+
+    def test_jacobian_create_graph_vectorize(self):
+        self._test_jacobian_create_graph(vectorize=True)
+
+    def _check_jacobian_vectorize_correctness(self, f, inputs):
+        expected = autogradF.jacobian(f, inputs)
+        result = autogradF.jacobian(f, inputs, vectorize=True)
+        self.assertEqual(result, expected)
+
+    def test_jacobian_vectorize_correctness_simple(self):
+        def f(x):
+            return 3 * x ** 2
+
+        x = torch.randn(2, 3, 5)
+        self._check_jacobian_vectorize_correctness(f, x)
+
+    def test_jacobian_vectorize_correctness_multi_input(self):
+        def f(x, y):
+            return (x.cos() * x) @ y.sin()
+
+        x = torch.randn(2, 3)
+        y = torch.randn(3, 5)
+        self._check_jacobian_vectorize_correctness(f, (x, y))
+
+    def test_jacobian_vectorize_correctness_multi_input_multi_output(self):
+        def f(x, y):
+            return (x * x) @ y, x @ (x.sum(1) * y)
+
+        x = torch.randn(5, 3)
+        y = torch.randn(3, 5)
+        self._check_jacobian_vectorize_correctness(f, (x, y))
+
+    def _check_hessian_vectorize_correctness(self, f, inputs):
+        expected = autogradF.jacobian(f, inputs)
```

Comment:
These should be `hessian`?

Reply:
Oops copy-paste error. Thanks for the catch!